partd/pandas.py

```diff
@@ -14,6 +15,10 @@
 
 dumps = partial(pickle.dumps, protocol=pickle.HIGHEST_PROTOCOL)
 
+_NUMPY = 'numpy_type'
+_CATEGORICAL = 'categorical_type'
+_DATETIME_WITH_TZ = 'datetime64_tz_type'
```

Comment:
Sorry, my previous comment was intended to mean "why not just inline `'numpy_type'` everywhere?